### PR TITLE
merge commit

### PR DESCRIPTION
**Pull request by nivethamei**
merge class done

### CODE REVIEW

**Comment by nivethamei** [comment]
merge done

File: AutomationFramework/src/test/java/versionControlTool/ToLearnMerge.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package versionControlTool;
+
+public class ToLearnMerge {
+
+	public static void main(String[] args) {
+	System.out.println("merge done");
```
Comment (rajabalaji53) [inline AutomationFramework/src/test/java/versionControlTool/ToLearnMerge.java:6]: done

**Comment by nivethamei** [comment]
https://github.com/nivethamei/framework.git

**Comment by rajabalaji53** [comment]
https://github.com/rajabalaji53/comvtigerCRM.git

**Pull request merged**